source/security/client-side-field-level-encryption-guide.txt

```diff
@@ -272,6 +272,23 @@ Additional Dependencies
          * - x64 Support
            - x64 support is required for CSFLE
 
+   .. tab::
+      :tabid: go
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Dependency Name
+           - Description
+
+         * - Go version 1.10+
+           - Version 1.10 or above is required
+
+         * - ``libmongocrypt`` 1.1.0+
```

Comment:
What do you think about linking libmongocrypt here instead of the other column, similar to how it's listed in Java?
That way, you don't have to mention it twice in the same row in adjacent columns.